src/lib/triggers.js

```diff
@@ -169,7 +165,7 @@ class Triggers {
 						const openFunc = () => {
 							const { user } = store.state;
 							if (user) {
-								if (trigger.runOnce) { store.off('change', openFunc); }
+								store.off('change', openFunc);
```

Comment:
Why removing the 'change' handler? 👀 (not familiar with the triggers code, just wanted to understand how this affects)

Reply:
Well, without this comments were being sent in an infinite loop :eyes: